DfE.FindInformationAcademiesTrusts/Program.cs

```diff
@@ -258,6 +264,27 @@ private static void AddAuthenticationServices(WebApplicationBuilder builder)
             });
     }
 
+    private static void AddApiHttpClient(WebApplicationBuilder builder)
+    {
+        builder.Services.AddHttpClient("DfE.FindInformationAcademiesTrusts.API", httpClient =>
+        {
+            var options = new FindInformationAcademiesTrustsApiOptions();
+
+            if (string.IsNullOrEmpty(options.BaseUri))
+            {
+                throw new ArgumentNullException("API Base Uri is not specified");
+            }
+
+            if (string.IsNullOrEmpty(options.ClientSecret))
+            {
+                throw new ArgumentNullException("Web App Client Secret is null or empty");
+            }
+
+            httpClient.BaseAddress = new Uri(options.BaseUri);
+            httpClient.DefaultRequestHeaders.Add("Accept", "application/json");
```

Comment:
Do we need to add the client secret here somewhere?

Reply:
The Client Secret will be used elsewhere :)